tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -109,14 +118,31 @@ def _MaybeSerialize(x):
 
 @beam.ptransform_fn
 @beam.typehints.with_input_types(beam.Pipeline)
-@beam.typehints.with_output_types(Union[tf.train.Example, bytes])
+@beam.typehints.with_output_types(Union[tf.train.Example,
+                                        tf.train.SequenceExample,
+                                        bytes])
```

Comment:
I wonder if we should drop _InputToExampleOrBytes, as it's basically the same as input_to_example

and could you also rename input_to_example to input_to_record

Reply:
should i just delete _InputToExampleOrBytes? or do we keep it for now?

Reply:
i deleted _InputToExampleOrBytes for now